### ISSUE TITLE
Sign events with any identity (private key)

### ISSUE THREAD

**Issue by tcheeric** [tcheeric/nostr-java#84]
Currently, you can only sign events with the identity configured in the profile.properties.
Make the necessary changes in nostr-api to provide a different identity for signing events

**Comment by tcheeric** [comment]
We create the concept of "custom identities", similar to the legacy identity.
A custom identity is identified by it's name, it has a corresponding configuration file where the private key is stored. The name of the properties file matches the name of the custom identity, and is stored in a folder defined in the app.properties.

**Comment by guilhermegps** [comment]
Has this issue been developed?

**Comment by tcheeric** [comment]
Yes. It should be in the 0.3-SNAPSHOT branch. Look for the CustomIdentity class...
-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


14 Nov 2023, 17:22 by ***@***.***:

**Comment by tcheeric** [comment]
https://github.com/tcheeric/nostr-java/blob/0.3-SNAPSHOT/nostr-java-id/src/main/java/nostr/id/CustomIdentity.java

-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


14 Nov 2023, 17:22 by ***@***.***:
